HW3/Author.java

```diff
@@ -0,0 +1,50 @@
+package HW3;
+
+public class Author {
+    int ID;
```

Comment:
all class fields and method attributes should start from lowercase

Reply:
updated in [5511bf4](https://github.com/naztekinalpp/NazTekinalp/pull/6/commits/5511bf40b8a9d54751805b39d256d7d91ea08ee8)